provisioning/osx/configure/src/nginx.js

```diff
@@ -0,0 +1,60 @@
+import execa from 'execa';
+import R from 'ramda';
+import fs from 'fs';
+
+const baseConfig = `server {
+  listen 80;
+  location / {
+    return 404;
+  }
+}
+`;
+
+export const create = (services, domain) => `
+${baseConfig}${R.join(
+  '',
+  R.map(
+    s => `
```

Comment:
Yeah I know it's a pile of mess. These things are never clean without using a proper templating library. Which I should probably have done. This file is deleted in https://github.com/redbadger/stack/pull/23 so won't bother to change